neurophotonics/fields.py

```diff
@@ -158,18 +158,18 @@ def plot(self, figsize=(8, 8), axis=None, gamma=0.7, cmap="magma", title=""):
         axis.set_title(title)
 
 
-@schema
-class Efield2dimage(dj.Computed):
-    definition = """
-    # Emission field images in 2D.
-    -> Esim
-    pov: smallint  # point of view direction (compression direction)
-    ---
-    projected_image: blob@photonics
-    """
-
-    def make(self, key):
-        volume = (EField & key).fetch1("volume")  # There will be only 1 volume per key.
-
-        for pov in [0, 1, 2]:
-            self.insert1(dict(key, pov=pov, projected_image=volume.sum(axis=pov)))
+# @schema
```

Comment:
Don't leave commented code. If you need old code, you can find it in the git history.